ui/docs/manual/using/administration.md

```diff
@@ -4,8 +4,6 @@ title: Project Administration
 order: 90
 ---
 
-# Project Administration
```

Comment:
Out of curiosity, what's the reason for removing this header? I think this is a top-level heading so there's no need to remove it. This comment applies to the other changes in the docs as well.

Reply:
Oh, I thought that was the rule: if there's only one top-level heading in the file, remove it as it's duplicative of the document title.

Reply:
Ah, I see your point. I think the problem is that not every top-level header share the same name as the filename. For example, a user viewing the login file won't necessarily know that it's docs about the login service. Another thing is that README.md files have no document title. We just use "Documentation" in those instances. Apologies for not bringing that up earlier.